<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighwayWindow</class>
 <widget class="QMainWindow" name="HighwayWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Highway</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <spacer name="horizontalSpacerLeft">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="roadLayout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacerRight">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="settingsLayout">
        <item>
         <widget class="QPushButton" name="addLaneButton">
          <property name="text">
           <string>Добавить полосу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteLaneButton">
          <property name="text">
           <string>Убрать полосу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCarButton">
          <property name="text">
           <string>Добавить автомобиль</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Скорость</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinMinSpeedBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>130</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinMaxSpeedBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>130</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
